composer.json

```diff
@@ -20,36 +20,24 @@
         }
     },
     "require": {
-        "php": ">=7.1",
-        "league/flysystem": "^1.0.40",
+        "php": "^7.2 || ^8.0",
+        "league/flysystem": "^2.0",
         "symfony/config": "^4.2|^5.0",
         "symfony/http-kernel": "^4.2|^5.0",
         "symfony/dependency-injection": "^4.2|^5.0",
         "symfony/options-resolver": "^4.2|^5.0"
     },
     "require-dev": {
         "async-aws/flysystem-s3": "^0.3",
-        "league/flysystem-aws-s3-v3": "^1.0.22",
-        "league/flysystem-azure-blob-storage": "^0.1.5",
-        "league/flysystem-cached-adapter": "^1.0.9",
-        "league/flysystem-memory": "^1.0",
-        "league/flysystem-rackspace": "^1.0",
-        "league/flysystem-replicate-adapter": "^1.0",
-        "league/flysystem-sftp": "^1.0",
-        "league/flysystem-webdav": "^1.0",
-        "league/flysystem-ziparchive": "^1.0",
+        "league/flysystem-aws-s3-v3": "^2.0",
+        "league/flysystem-memory": "^2.0",
+        "lustmored/flysystem-v2-simple-cache-adapter": "^0.1.0",
```

Comment:
There is no cache adapter anymore? Perhaps would it make sense to remove the adapter then?

Reply:
There is no upstream adapter for v2 and there seems to be no work towards it. V1 adapter also was very memory inefficient (https://github.com/thephpleague/flysystem-cached-adapter/issues/23). That is why I have created this simple alternative for v2, which is very helpful in my projects already, so I think it can be helpful for others too.

But of course if you feel like dropping it from the bundle, I can do so.

Reply:
Let's drop it for now, we'll add it if someone find it useful (not sure it is)

Reply:
OK, I have removed it from code and docs